Library/Homebrew/utils/analytics.rb

```diff
@@ -36,7 +36,7 @@ def report_google(type, metadata = {})
             --data aip=1
             --data t=#{type}
             --data tid=#{analytics_id}
-            --data cid=#{ENV.fetch("HOMEBREW_ANALYTICS_USER_UUID")}
+            --data uid=n0thxg00gl3
```

Comment:
Have to specify a `cid` (UUID) or `uid` (any string) here so: picked this one and it works as expected locally.